pages/jobbannonse.js

```diff
@@ -1,23 +1,39 @@
+//@flow
 import { useEffect } from 'react';
 import Router from 'next/router';
+import { graphql, type Variables } from 'react-relay';
+import type { NextRouter } from '../utils/types';
+import { withData } from '../lib/withData';
 
 // In order to keep backward compatibility urls like /jobbannonse?id=zyx
 
 // For client redirects
-const Redirect = props => {
+const Redirect = () => {
   useEffect(() => {
-    Router.replace(props.newLocation);
-  }, [props.newLocation]);
+    Router.replace('/jobb');
+  }, []);
   return null;
 };
```

Comment:
So no redirect on the client now? (not that it's necessary)

Reply:
Are there still any old links that use shallow routing on itdagene.no? If they are present in articles etc. they will not be not use shallow routing anyway, so it shouldn't matter.